index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Resume Builder</title>
+    <link rel="stylesheet" href="./styles/style.css">
+</head>
+
+<body class="wrapper">
+    <header>
+        <h1>JSON Resume Builder</h1>
+    </header>
+    <main>
+        <section class="builder">
+            <form class="builderTemplate">
+                <h2>Input</h2>
+                <label for="userInput">Complete template below:</label>
```

Comment:
I like the straightforwardness of this approach. I am uncertain about it because once the user fills out the template in this manner, they will already have a JSON object. At least if I'm understanding correctly. The input is text that looks exactly like the output.

Reply:
I love the overall design.  I have to agree with @JWaltuch; I thought the requirements were to take text or a string via an input filed and convert it to JSON.  It looks like with your approach the user would already have the data in a JSON object.

Reply:
😆 it's true, I took the simplest approach that may need a lot more error handling or built in linting if it was ever actually used. Thank you for the comments!